src/WebApi/Extensions/ErrorOrExt.cs

```diff
@@ -0,0 +1,49 @@
+using ErrorOr;
+using Microsoft.AspNetCore.Http.HttpResults;
+
+namespace SSW.CleanArchitecture.WebApi.Extensions;
+
+public static class ErrorOrExt
```

Comment:
```suggestion
public static class ErrorOrExtensions
```

Prefer something a bit more verbose, we don't have a character restriction here

Reply:
I ended up renaming to `CustomResult`.  Originally this was an extension method, but is not anymore.